client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/product/DomainsTransferredList.tsx

```diff
@@ -1,17 +1,21 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
+import formatCurrency from '@automattic/format-currency';
+import { joinClasses } from '@automattic/wpcom-checkout';
+import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
 import { connect } from 'react-redux';
 import { domainManagementRoot, domainManagementTransferIn } from 'calypso/my-sites/domains/paths';
 import { getSiteSlug } from 'calypso/state/sites/selectors';
-import './style.scss';
 import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+import './style.scss';
 
 type Props = {
 	purchases: ReceiptPurchase[] | undefined;
 	manageDomainUrl: string;
+	currency?: string;
 };
 
-const DomainsTransferredList = ( { purchases, manageDomainUrl }: Props ) => {
+const DomainsTransferredList = ( { purchases, manageDomainUrl, currency = 'USD' }: Props ) => {
```

Comment:
Are we rendering this without data? Curious why we need a default currency.

Reply:
I think before it looked like this: https://github.com/Automattic/wp-calypso/pull/79755, so it didn't have any currency data.

![image](https://user-images.githubusercontent.com/1126811/255300046-a60b5930-4ab9-4cbe-9629-71a48b176e63.png)

Reply:
> Are we rendering this without data? Curious why we need a default currency.

I misunderstood the comment. Have the same question as well.

Reply:
> The formatCurrency function has a default fallback value to USD as well.

I don't like any of the USA/English fallbacks we have but I suspect this one is just for consistency with the way we fallback on English for get_locale calls.

In this case I'm just wondering why we need the fallback, if we have a receipt we should always have a currency.